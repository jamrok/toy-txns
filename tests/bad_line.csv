type,        client,  tx,  amount
deposit,     3,       1,   5.4321
cashback,    3,       5,   10.0
deposit,     2,       2,   2.0
dispute,     3,       2,
withdrawal,  3,       2,   2.0
