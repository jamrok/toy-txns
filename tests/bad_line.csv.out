client,available,held,total,locked
2,2,0,2,false
3,3.4321,0,3.4321,false
